stackit/internal/services/logme/logme_acc_test.go

```diff
@@ -62,7 +91,13 @@ func TestAccLogMeResource(t *testing.T) {
 
 			// Creation
 			{
-				Config: resourceConfig(instanceResource["sgw_acl-1"]),
+				Config: resourceConfig(
+					map[string]string{
+						"sgw_acl":            instanceResource["sgw_acl-1"],
+						"fluentd_tcp":        instanceResource["fluent_tcp"],
+						"max_disk_threshold": instanceResource["max_disk_threshold"],
+						"enable_monitoring":  instanceResource["enable_monitoring"],
+					}),
```

Comment:
Wouldn't it be better to make sure that all parameters work? Although I believe there are some parameters that we don't know valid values for, I think we can exclude those

Reply:
I'm extend it with parameters as I find out some values that work, but creation and update of the instance is taking almost 1 hour

Reply:
Added a `syslog` list field for type completion

Reply:
And the `ism_jitter` float parameter